src/configure.ts

```diff
@@ -4,22 +4,30 @@ import { exec } from 'child_process';
 
 import { CACHE_FOLDER } from './constants';
 import { createCacheFolderIfNeeded } from './utils';
+import { debug } from './preview';
 
 interface JestPreviewConfigOptions {
-  externalCss: string[];
+  externalCss?: string[];
+  autoPreview?: boolean;
   publicFolder?: string;
 }
 
-export async function jestPreviewConfigure(
-  options: JestPreviewConfigOptions = { externalCss: [] },
-) {
+export async function jestPreviewConfigure({
+  externalCss = [],
+  autoPreview = true,
+  publicFolder,
+}: JestPreviewConfigOptions) {
```

Comment:
@nvh95 I think we should set a default options object here, something like

```ts
export async function jestPreviewConfigure({
  externalCss = [],
  autoPreview = true,
  publicFolder = 'public',
}: JestPreviewConfigOptions = { externalCss: [], autoPreview: true, publicFolder: 'public' }) {
```

This syntax is not the prettiest, but if we don't set a default param, this function call will result in a TypeError if nothing is passed in.

```ts
jestPreviewConfigure(); // This will throw
```

Also as per the doc below, the default for `publicFolder` should be `public`

Reply:
> I think we should set a default options object here

Sure. I didn't notice that. Thanks.



> Also as per the doc below, the default for `publicFolder` should be `public`

Thanks. If users' public folder is `public`, the code in `jestPreviewConfigure` shouldn't be executed. That's intentional to avoid writing unnecessary File I/O operations. Default users' folder is defined in `previewServer.js`. I'm going to update the docs.
https://github.com/nvh95/jest-preview/blob/79b5b747156734551e006b6e6a9fd118b3ce3b1d/server/previewServer.js#L31-L32